vspace/pom.xml

```diff
@@ -12,6 +12,9 @@
         <org.slf4j-version>1.7.5</org.slf4j-version>
         <spring.version>5.3.0</spring.version>
         <spring-data.version>Kay-SR8</spring-data.version>
+        <spring-security-version>5.3.10.RELEASE</spring-security-version>
```

Comment:
`spring-security-version` is defined in line 18. what's the right version?

Reply:
The version was 5.5.7 for spring-security-version